### DISCUSSION TITLE
Unknown arg `id` in ... for type "CreateNestedOneWithout...Input"

### DISCUSSION THREAD

**Discussion by arioberek** [prisma/prisma#12216] [Q&A] [answered]
I'm having this issue while trying to post an object to Prisma (on prisma.create):

```
Unknown arg `id` in data.office_key.id for type OfficeKeyCreateNestedOneWithoutComputerInput. Available args:

type OfficeKeyCreateNestedOneWithoutComputerInput {
  create?: OfficeKeyCreateWithoutComputerInput | OfficeKeyUncheckedCreateWithoutComputerInput
  connectOrCreate?: OfficeKeyCreateOrConnectWithoutComputerInput
  connect?: OfficeKeyWhereUniqueInput
}

```

What am I doing wrong? Should I post only the id of the model's relation?

**Comment by austincrim** [comment, marked as answer]
Hey @arrbrk1 👋 ,

I could confirm this if you shared the erroring code snippet, but it seems like you are doing a [nested write](https://github.com/prisma/prisma/discussions/12210) but not specifying the operation for `office_key`.

```ts
await prisma.some_model.create({
  data: {
    office_key: {
      create: {   // must specify `create`, `connect`, or `connectOrCreate`
        id: 'some id'
      }
    }
  }
})
``` 

See the link above for more examples.
Let me know if I can clarify!

**Comment by joshribakoff-sm** [comment]
I got the same error, and it was due to passing Prisma invalid inputs:

```
    industry: {
    ~~~~~~~~
      connect: {
        id: NaN
      }
    },
```
